docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_QuartzNet.md

```diff
@@ -6,15 +6,23 @@
 
 To download the pre-trained model, refer to the [NeMo Speech Models Catalog](https://ngc.nvidia.com/catalog/models/nvidia:nemospeechmodels).
 Here are the instructions on how to obtain QuartzNet in ONNX* format.
+
+Install the NeMo toolkit using [instructions](https://github.com/NVIDIA/NeMo/tree/main#installation).
```

Comment:
Install the NeMo toolkit using [instructions] ---> 1. Install the NeMo toolkit using the [instructions]